FirebaseAuth/Sources/Utilities/FIRAuthErrorUtils.m

```diff
@@ -615,7 +621,9 @@
     @"The request is not supported or is invalid.";
 
 static NSString *const kFIRAuthErrorMessageRecaptchaSDKNotLinked =
-    @"The recaptcha SDK is not linked to your app.";
+    @"The recaptcha SDK is not linked to your app. Please follow "
+    @"https://cloud.google.com/recaptcha-enterprise/docs/instrument-ios-apps to add Recaptcha "
```

Comment:
Ideally we should point them to GCIP docs, which in turn point to this doc - https://cloud.google.com/recaptcha-enterprise/docs/instrument-ios-apps